<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProjectSettings</class>
 <widget class="QDialog" name="DialogProjectSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabSettings">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Selected music file:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Music file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSongFile">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBrowse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Project lyrics type</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="rbLRC1">
            <property name="text">
             <string>LRC version 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbLRC2">
            <property name="text">
             <string>LRC version 2</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbLRC3">
            <property name="text">
             <string>UltraStar lyrics</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbLRC4">
            <property name="text">
             <string>CD+G lyrics</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRequired">
      <attribute name="title">
       <string>Required fields</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>All those fields must be set</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Song title:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leTitle"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Artist:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leArtist"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelMusicFile">
            <property name="text">
             <string>Music file:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leUSmp3"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="labelUSbpmgap">
            <property name="text">
             <string>BPM and GAP will be added by the editor</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>315</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOptional">
      <attribute name="title">
       <string>Optional fields</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupLRC">
         <property name="title">
          <string>LRC format optional fields</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Album</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leLRCalbum"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Created by:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leLRCcreated"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Offset:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leLRCoffset"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Application:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leLRCapp"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leLRCappver"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupUStar">
         <property name="title">
          <string>Ultrastar format optional fields</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Video file:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leUSvideo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Video gap:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leUSvideogap"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Cover image:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leUScover"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Background image:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leUSbackground"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Genre:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leUSgenre"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Edition:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="leUSedition"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Language:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="leUSlang"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCDG">
      <attribute name="title">
       <string>CD+G output</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupCDG">
         <property name="title">
          <string>CD+G rendering</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Font:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="fontCDG">
              <property name="whatsThis">
               <string>Font family for preview window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="fontCDGSize">
              <property name="whatsThis">
               <string>Font size for preview window. Larger values are suggested (24 and up)</string>
              </property>
              <property name="minimum">
               <number>4</number>
              </property>
              <property name="maximum">
               <number>120</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Keep title at least </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="titleCDGmin">
              <property name="maximum">
               <number>120</number>
              </property>
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>seconds</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Colors</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Backround</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="ColorButton" name="btnCDGColorBg">
               <property name="whatsThis">
                <string>Preview window background color.</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Information</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="ColorButton" name="btnCDGColorInfo">
               <property name="whatsThis">
                <string>Preview window text color for the text which has been sung.</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Sung</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ColorButton" name="btnCDGColorInactive">
               <property name="whatsThis">
                <string>Preview window text color for the text which is not being sung yet..</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Not sung yet</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="ColorButton" name="btnCDGColorActive">
               <property name="whatsThis">
                <string>Preview window text color for the text which has been sung.</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbCDGPreamble">
            <property name="text">
             <string>Draw squares showing where singing starts</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>238</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogProjectSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogProjectSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
